doc/material-getting-started.md

```diff
@@ -10,14 +10,64 @@ Uno Material lets you apply [Material Design 3](https://m3.material.io/) styling
+
+### `MaterialTheme`
+
+There is now a new, simplified method for initializing the Material Toolkit resources. It is recommended to use the new `MaterialTheme` going forward since `MaterialResources`, `MaterialColors`, and `MaterialFonts` have been deprecated.
+
+#### Inheritance
+ResourceDictionary &#8594; MaterialTheme
+
+#### Constructors
+| Constructor    | Description                                           |
+|----------------|-------------------------------------------------------|
+| MaterialTheme()         | Initializes a new instance of the `MaterialTheme` `ResourceDictionary`.       |
+| MaterialTheme(`ResourceDictionary?` colorOverride, `ResourceDictionary?` fontOverride)         | Initializes a new instance of the `MaterialTheme` `ResourceDictionary` and applies the given overrides       |
+
+#### Properties
+
+| Property                  | Type              | Description                                                                                   |
+|---------------------------|-------------------|-----------------------------------------------------------------------------------------------|
+| ColorOverrideSource             | string            | (Optional) Gets or sets a Uniform Resource Identifier that provides the source location of a ResourceDictionary containing overrides for the default Uno.Material Color resources                                            |
+| FontOverrideSource     | string      | (Optional) Gets or sets a Uniform Resource Identifier that provides the source location of a ResourceDictionary containing overrides for the default Uno.Material FontFamily resources            |
+
+> [!NOTE]
+> As of [Uno 4.7](https://platform.uno/blog/uno-platform-4-7-new-project-template-performance-improvements-and-more/), the solution template of an Uno app has changed. There is no longer a Shared project (.shproj), it has been replaced with a regular cross-platform library containing all user code files, referred to as the **App Code Library** project. This also implies that package references can be included in a single location without the previous need to include those in all project heads.
+
+The following installation steps are based on the new Uno solution template structure.
+
+1. Create a new Uno project using the `Uno Platform App` template.
+2. In the Solution Explorer panel, right-click on your app's **App Code Library** project (`PROJECT_NAME.csproj`) and select `Manage NuGet Packages...`
+3. Install the [`Uno.Material.WinUI`](https://www.nuget.org/packages/Uno.Material.WinUI)
+4. Add the following Material resources `AppResources.xaml`:
+
+	```xml
+	<ResourceDictionary>
+		<ResourceDictionary.MergedDictionaries>
+
+			<!-- Load Uno.Material resources -->
+			<MaterialTheme xmlns="using:Uno.Material" />
+
+		</ResourceDictionary.MergedDictionaries>
+	</ResourceDictionary>
+	```
+
+### Previous Installation Method
```

Comment:
```suggestion
### Installing Uno.Material on previous versions of Uno Platform
```

While technically not true, because it's not exactly related to the uno version, we can associate the two.